pkgs/desktops/cinnamon/xapps/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub, glib, gobject-introspection, gtk3, libgnomekbd, libxklavier, meson, ninja, pkgconfig, python3, python3Packages, stdenv, vala, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "xapps";
+  version = "1.6.4";
+  enableParallelBuilding = true;
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "08c8j7wv56zf8mfgjfhd0wc1bbrw0dj1lmlawvsxwfajnnyjsm0d";
+  };
+
+  NIX_CFLAGS_COMPILE = [ "-I${glib.dev}/include/gio-unix-2.0" ]; # TODO: https://github.com/NixOS/nixpkgs/issues/36468
```

Comment:
I think I can submit a patch to fix this

Reply:
Go for it ^^

Reply:
I am not certain this is more future proof – it does not carry out semantic difference and it only silently falls back to `out`, it does not even respect output redirection. We would need something like `getInclude = pkg: if pkg ? outputInclude then pkg.${pkg.outputInclude} else pkg.include or getDev pkg`.

Reply:
Ideally this would be patched away in the future

Reply:
Should we leave it as-is then and patch it away upstream?

Reply:
I'm not sure this is pertinent to the review anymore.